.editorconfig

```diff
@@ -4,6 +4,7 @@ root = true
 indent_style = space
 indent_size = 2
 tab_width = 2
+end_of_line = lf
```

Comment:
I'm using Windows, and oddly when I open some files in Atom, they have CRLF line endings. I have it set up to always save with unix line endings... this addition shouldn't cause any issues.